### PR TITLE
Very cool animation for lists done with two lines on code

### PR DESCRIPTION
**Pull request by T-Damer**

### CODE REVIEW

File: src/components/proofs/ContractListContainer.tsx [hunk 1]
```diff
@@ -8,5 +10,10 @@ const container = classnames(
 )
 
 export default function ({ children }: ChildrenProp) {
-  return <div className={container}>{children}</div>
+  const listRef = useAutoAnimate({})
+  return (
+    <div className={container} ref={listRef as Ref<HTMLDivElement>}>
```
Comment (smyzin) [inline src/components/proofs/ContractListContainer.tsx:15]: Can't we do `const listRef = useAutoAnimate<HTMLDivElement>({})`?

File: src/components/badges/BadgeSection.tsx [hunk 2]
```diff
@@ -30,10 +31,11 @@ export default function ({
   onMinted?: () => void
 }) {
   if (minted.length === 0 && proofs.length === 0) return null
+  const listRef = useAutoAnimate({})
```
Comment (smyzin) [inline src/components/badges/BadgeSection.tsx:34]: Also, is it possible to set it as `useAutoAnimate<HTMLDivElement>`?

Comment (T-Damer) [reply src/components/badges/BadgeSection.tsx:34]: It doesnt eat typings

File: src/components/proofs/ContractListContainer.tsx (hunk 1 above)
Comment (T-Damer) [reply src/components/proofs/ContractListContainer.tsx:15]: No, we can't, it doesn't accept type notations

**Review by smyzin — APPROVED** [review]
Lgtm

File: src/components/badges/BadgeSection.tsx [hunk 3]
```diff
@@ -30,10 +31,11 @@ export default function ({
   onMinted?: () => void
 }) {
   if (minted.length === 0 && proofs.length === 0) return null
```
Comment (MixailE) [inline src/components/badges/BadgeSection.tsx:33]: This return will not lead to errors?

Comment (T-Damer) [reply src/components/badges/BadgeSection.tsx:34]: Fixed it

**Pull request merged**